src/dialogsearch.cpp

```diff
@@ -21,6 +22,13 @@ DialogSearch::DialogSearch(QWidget *parent) :
     ui->list_result->setSelectionBehavior(QAbstractItemView::SelectRows);
     ui->list_result->setEditTriggers(QAbstractItemView::NoEditTriggers);
 
+
+    for (auto *box : {ui->combo_variable, ui->combo_switch, ui->combo_item, ui->combo_eventname})
+    {
+        box->completer()->setFilterMode(Qt::MatchContains);
+        box->completer()->setCompletionMode(QCompleter::PopupCompletion);
+    }
+
```

Comment:
No, this doesn't make it better, when another groupBox is added you have to still remember to add a line there >.>